src/components/misc/CopyText.vue

```diff
@@ -18,7 +18,8 @@ export default {
             let copytext = this.$refs.copytext
             copytext.select()
             copytext.setSelectionRange(0, 99999)
-
+            let { dispatchNotification } = this.globalHelper()
+            dispatchNotification({ message: 'Copied to clipboard', type: 'success' })
```

Comment:
Please use translation keys in wallet

Reply:
@aeddaqqa And also unify the wording letter case (Upper and lower case).

Reply:
done